.editorconfig

```diff
@@ -0,0 +1,28 @@
+# EditorConfig for MakerDAO Projects
+
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+charset = utf-8
+
+[*.{sol,sh,js,py}]
+indent_style = space
+indent_size = 4
+
+[*.{diff,md}]
+trim_trailing_whitespace = false
+
+[{package.json,.travis.yml}]
+indent_style = space
+indent_size = 2
+
+[foundry.toml]
+indent_style = space
+indent_size = 4
```

Comment:
needs new line at end of file (really would have hoped that line 11 in this file would have taken care of that...)